### PR TITLE
Port System and pkg/lib to PF4-React button, some code cleanup

### PR DESCRIPTION
**Pull request by martinpitt**

### CODE REVIEW

**Comment by martinpitt** [comment]
I visually tested all the buttons and dialogs, they look exactly the same to me as before.

**Comment by martinpitt** [comment]
Nice, this [works now](https://logs.cockpit-project.org/logs/pull-13670-20200306-130528-a61dda60-fedora-32/log.html). Triggering all tests.

File: pkg/systemd/reporting.jsx [hunk 1]
```diff
@@ -413,20 +414,20 @@ function WorkflowRow(props) {
     let button = null;
     if (props.problemState === ProblemState.REPORTING) {
         button = (
-            <button key={"cancel_" + props.label}
-                    className="pf-c-button pf-m-danger"
+            <Button key={"cancel_" + props.label}
```
Comment (KKoukiou) [inline pkg/systemd/reporting.jsx]: Nothing related to this PR - but cancel buttons should not be danger I think.

Comment (martinpitt) [reply pkg/systemd/reporting.jsx]: Good point, sent https://github.com/cockpit-project/cockpit/pull/13684

**Pull request merged**